doc/source/dev/governance/governance.rst

```diff
@@ -0,0 +1,376 @@
+of the Project Software and Services that Council Members will provide
+useful guidance, both technical and in terms of project direction, to
+potentially less experienced contributors.
+
+The Steering Council and its Members play a special role in certain
+situations. In particular, the Council may, if necessary:
+
+-  Make decisions about the overall scope, vision and direction of the
+   project.
+-  Make decisions about strategic collaborations with other
+   organizations or individuals.
+-  Make decisions about specific technical issues, features, bugs and
+   pull requests. They are the primary mechanism of guiding the code
+   review process and merging pull requests.
+-  Make decisions about the Services that are run by The Project and
+   manage those Services for the benefit of the Project and Community.
+-  Update policy documents such as this one.
+-  Make decisions when regular community discussion doesn’t produce
+   consensus on an issue in a reasonable time frame.
+
+However, the Council's primary responsibility is to facilitate the
+ordinary community-based decision making procedure described above. If
+we ever have to step in and formally override the community for the
+health of the Project, then we will do so, but we will consider reaching
+this point to indicate a failure in our leadership.
+
+Council decision making
+~~~~~~~~~~~~~~~~~~~~~~~
+
+If it becomes necessary for the Steering Council to produce a formal
+decision, then they will use a form of the `Apache Foundation voting
+process <https://www.apache.org/foundation/voting.html>`_. This is a
+formalized version of consensus, in which +1 votes indicate agreement,
+-1 votes are vetoes (and must be accompanied with a rationale, as
+above), and one can also vote fractionally (e.g. -0.5, +0.5) if one
+wishes to express an opinion without registering a full veto. These
+numeric votes are also often used informally as a way of getting a
+general sense of people's feelings on some issue, and should not
+normally be taken as formal votes. A formal vote only occurs if
+explicitly declared, and if this does occur then the vote should be held
+open for long enough to give all interested Council Members a chance to
+respond -- at least one week.
+
+In practice, we anticipate that for most Steering Council decisions
+(e.g., voting in new members) a more informal process will suffice.
+
+Council membership
+~~~~~~~~~~~~~~~~~~
+
+To become eligible to join the Steering Council, an individual must be a
+Project Contributor who has produced contributions that are substantial
+in quality and quantity, and sustained over at least one year. Potential
+Council Members are nominated by existing Council members and voted upon
+by the existing Council after asking if the potential Member is
+interested and willing to serve in that capacity. The Council will be
+initially formed from the set of existing Core Developers who, as of
+late 2015, have been significantly active over the last year.
```

Comment:
Significantly is too vague, I think. Original attempt was commit bit, second was committed a merge (?), which both had the advantage of being public and concrete.